Expensify/k2-extension
# Show unfiltered issues at the top of the dashboard

Fixes a bug where not all issues were shown on the dashboard.

Fixes https://github.com/Expensify/Expensify/issues/219996

# Tests
1. Load the dashboard
2. Verify all your assigned issues are listed at the top
